docker-compose.yaml

```diff
@@ -55,7 +55,9 @@ services:
       - "30333:30333"
     volumes:
       - ./spec-data:/cereio/spec-data
-      - ./chain-data:/data
+      - type: volume
+        source: chainData
```

Comment:
Can we use kebab case here? If so, let's do this everywhere.

Reply:
Yes, updated with kebab case